include/lib/allocation_tracker.hpp

```diff
@@ -63,8 +63,9 @@ class AllocationTracker {
   static TrackerThreadLocalState *get_tl_state();
 
 private:
-  using AdressSet = std::unordered_set<uintptr_t>;
+  using AddressSet = std::unordered_set<uintptr_t>;
```

Comment:
apologies for the typo, this is sadly going to conflict with #298 where I redefined it as `AddressBitset _allocated_address_set;`